cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -1277,3 +1277,30 @@ func convertUnstructuredKhCheck(unstructured unstructured.Unstructured) (khcheck
 
 	return khCheck, err
 }
+
+func watchUnstructuredKHChecks() (watch.Interface, error) {
+
+	var watcher watch.Interface
+	restConfig, err := clientcmd.BuildConfigFromFlags("", configPath)
+	if err != nil {
+		return watcher, err
+	}
+
+	dynClient, err := dynamic.NewForConfig(restConfig)
```

Comment:
Can we somehow re-use the single global Kubernetes client?  We had an issue long ago that stemmed from using multiple Kubernetes clients concurrently.

[this client](https://github.com/Comcast/kuberhealthy/blob/master/cmd/kuberhealthy/main.go#L78) is what I am proposing is used instead